src/managementgroup/azext_managementgroup/generated/_help.py

```diff
@@ -0,0 +1,241 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['managementgroup'] = '''
+    type: group
+    short-summary: Manage Management Groups API
+'''
+
+helps['managementgroup management-group'] = """
+    type: group
+    short-summary: Manage management group with managementgroup
+"""
+
+helps['managementgroup management-group list'] = """
+    type: command
+    short-summary: "List management groups for the authenticated user."
+    examples:
+      - name: ListManagementGroups
+        text: |-
+               az managementgroup management-group list
+"""
+
+helps['managementgroup management-group show'] = """
+    type: command
+    short-summary: "Get the details of the management group."
+    examples:
+      - name: GetManagementGroup
```

Comment:
It would be better if we could change it into a correct sentence format, such as:
```
Get management group
```